includes/core/classes/class-setup.php

```diff
@@ -64,6 +64,7 @@ protected function instantiate_classes(): void {
 		Event_Setup::get_instance();
 		Export::get_instance();
 		Import::get_instance();
+		Multisite::get_instance();
```

Comment:
I think this might be able to be worked into `Settings` singleton since it is directly related to that.